Lib/test/test_pdb.py

```diff
@@ -3235,6 +3237,47 @@ def test_checkline_is_not_executable(self):
                 self.assertFalse(db.checkline(os_helper.TESTFN, lineno))
 
 
+@support.requires_subprocess()
+class PdbTestReadline(unittest.TestCase):
+    def setUpClass():
+        # Ensure that the readline module is loaded
+        # If this fails, the test is skipped because SkipTest will be raised
+        readline = import_module('readline')
+        if readline.__doc__ and "libedit" in readline.__doc__:
+            raise unittest.SkipTest("libedit readline is not supported for pdb")
+
+    def test_basic_completion(self):
+        script = textwrap.dedent("""
+            import pdb; pdb.Pdb().set_trace()
+            print('hello')
+        """)
+
+        input = b"co\t\tntin\t\n"
```

Comment:
This is hard to follow, I suggest adding a comment.
```suggestion
        # List everything starting with 'co',
        # then complete and enter 'continue'
        input = b"co\t\tntin\t\n"
```